client/assets/css/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-image: url('images/homepage-image.png');
+    background-size: cover;
+    background-position: center;  
+    background-repeat: no-repeat;
+    min-height: 100vh;
+    background-attachment: fixed;
+    margin: 0;
+}
+
+nav {
+    display: grid;
+    background-color: #2C53DA;
+    grid-template-columns: 30% 70%;
+    border: 1px solid rgba(0, 0, 0, 0.8);
+    width: 100%;
+}
+
+#h4 {
```

Comment:
Using the id selector is quite specific here.  Can you use the element selector instead?  Would a class selector be more appropriate?

Reply:
I'm only assigning properties to this specific part of the html body, so I thought class would be needless. Also, I need to prioritize the properties assigned here over properties assigned to the two containers making up the outermost grid of the navbar. Using the element selector assigns the wrong properties to this tag.
![Screenshot 2021-07-05 at 19 43 41](https://user-images.githubusercontent.com/78097320/124509168-5e963780-ddc9-11eb-8646-f6ca8e44395c.png)